src/widgets/totalHrsAndGranteeGraph.js

```diff
@@ -38,10 +38,18 @@ function addOrUpdateResponse(traceIndex, res, xValue, valueToAdd) {
 export default async function totalHrsAndGranteeGraph(scopes, query) {
   // Build out return Graph data.
   const res = [
-    { name: 'Grantee Rec TTA', x: [], y: [] },
-    { name: 'Hours of Training', x: [], y: [] },
-    { name: 'Hours of Technical Assistance', x: [], y: [] },
-    { name: 'Hours of Both', x: [], y: [] },
+    {
+      name: 'Grantee Rec TTA', x: [], y: [], month: null,
+    },
```

Comment:
"Grantee Rec TTA" shows up in the accessible table but not the graph, is that expected?

Reply:
😱